nselib/netbios.lua

```diff
@@ -445,7 +445,7 @@ function nbquery(host, nbname, options)
         return false, "ERROR: Response contained no answers"
       end
       local dname = string.char(#resp.output.answers[1].dname) .. resp.output.answers[1].dname
-      table.insert( results, { peer = resp.peer, name = name_decode(dname) } )
+      table.insert( results, { peer = resp.peer, name = name_decode(dname), data = resp.output.answers[1].data } )
```

Comment:
Actualy I don't know another way to get raw-data response via `nbquery()` function.

Reply:
What I am asking here is why you have added the `data` field only here but not to the logically same output 5 lines lower. Not doing so is making the output of `netbios.nbquery` inconsistent.

Any particular reason why you have chosen to call `netbios.nbquery` with `{ multiple = true }` in the first place?

Reply:
Understood. I didn't not notice about 453 line. Fixed.
About `multiple` I just tried to find the most easy way to getting raw netbios response. It seems it can works without `multiple` argument. Actually I don't know any differences.

Reply:
Just FYI, option `multiple` is flagging the query packet as a NetBIOS broadcast, which means that the request could possibly receive *multiple* responses, which is not the objective in this case. You want to query the target and nobody else.